### PR TITLE
Adding mainLayout state

### PR DESCRIPTION
**Pull request by MayuBen1027**
Adding the state mainLayout based on the information found [here](https://github.com/inkdropapp/docs/blob/master/src/pages/reference/state-main-layout.md).

### CODE REVIEW

File: src/app/states/main-layout/page.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+export const metadata = {
+  title: 'main-layout',
```
Comment (Keisir) [inline src/app/states/main-layout/page.mdx:2]:
```suggestion
  title: 'mainLayout',
```

File: src/app/states/main-layout/page.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  description: 'An app state of the main layout. This section explains how to fetch the `mainLayout` state.',
+}
+
+# main layout
```
Comment (Keisir) [inline src/app/states/main-layout/page.mdx:6]:
```suggestion
# mainLayout
```

File: src/app/states/main-layout/page.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+---
+
+## Data Structure
+
+<Row>
+  <Col>
+    <Properties>
+      <Property name="distractionFreeEnabled" type="boolean">
+        true if distraction-free mode is enabled
```
Comment (Keisir) [inline src/app/states/main-layout/page.mdx:20]:
```suggestion
        `true` if distraction-free mode is enabled
```

File: src/app/states/main-layout/page.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      </Property>
+      <Property name="sidebarVisible" type="boolean">
+        true if side bar is visible
```
Comment (Keisir) [inline src/app/states/main-layout/page.mdx:23]:
```suggestion
        `true` if side bar is visible
```

**Review by Keisir — COMMENTED** [review]
Looks good!
I left some comments to keep the page similar to the others.

**Review by Keisir — APPROVED** [review]
Thank you, that looks good too!

**Comment by craftzdog** [comment]
🙌 Thank you

**Pull request merged**